src/ui.data-table/UTable.vue

```diff
@@ -607,6 +607,18 @@ function getDateSeparatorLabel(separatorDate) {
     : separatorDate;
 }
 
+function getDateSeparatorRowAttrs(rowIndex) {
+  const isPreviousRowChecked = tableRows.value[rowIndex - 1]?.isChecked;
+  const isCheckedRowAfter = tableRows.value[rowIndex]?.isChecked;
+  const isFirstRowAndNextChecked = rowIndex === 0 && isCheckedRowAfter;
+
+  (isPreviousRowChecked && isCheckedRowAfter) || isFirstRowAndNextChecked ? true : false;
+}
+
+function getRowAttrs(row) {
+  selectedRows.value.includes(row.id) ? true : false;
```

Comment:
`true : false` no needed.

`return` is needed.

`return Boolean(selectedRows.value.includes(row.id));`